models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1617,7 +1617,9 @@ FROM
     ('grain-granary','ethereum','GRAIN','0xf88baf18fab7e330fa0c4f83949e23f52fececce', 18),
     ('verse-verse-token', 'ethereum', 'VERSE', '0x249ca82617ec3dfb2589c4c17ab7ec9765350a18', 18),
     ('mc-merit-circle', 'ethereum', 'MC', '0x949d48eca67b17269629c7194f4b727d4ef9e5d6', 18),
-    ('r-r-stablecoin', 'ethereum', 'R', '0x183015a9ba6ff60230fdeadc3f43b3d788b13e21', 18)
+    ('r-r-stablecoin', 'ethereum', 'R', '0x183015a9ba6ff60230fdeadc3f43b3d788b13e21', 18),
+    ('crvusd-curvefi-usd-stablecoin', 'ethereum', 'CRVUSD', '0xf939e0a03fb07f59a73314e73794be0e57ac1b4e', 18),
+    ('reth-stafi', 'ethereum', 'RETH', '0x9559aaa82d9649c7a7b220e7c461d2e74c9a3593', 18)
```

Comment:
this `token_id` is inactive on coinpaprika, see here:
https://api.coinpaprika.com/v1/coins/reth-stafi

you can go to coinpaprika site and look to modify a token, they take in requests to activate. you could explore doing that? otherwise, this token will potentially get historical prices but then nothing moving forward

Reply:
you can also remove this from the PR in order to get the curve one merged